src/quacc/calculators/espresso/core.py

```diff
@@ -0,0 +1,48 @@
+from quacc.calculators.espresso.utils import construct_namelist
+from quacc.calculators.espresso.io import _write, _read
+from quacc.utils.files import load_yaml_calc
+
+# Very simple class that can run calculations ASE style
+# I don't think we even need atoms object here.
+class EspressoCalculator:
```

Comment:
You'll probably want to come up with a slightly clearer name for this so as to distinguish it from the main ASE calculator. Not quite sure yet.